### PR TITLE
Clean test scripts using as much as possible the opfab cli

### PR DESCRIPTION
**Pull request by freddidierRTE**
Nothing in release note

## Summary by CodeRabbit

- **New Features**
	- Introduced streamlined commands for loading configurations and bundles in the OperatorFabric environment.
	- Added individual commands for creating perimeter files and loading various process groups and business data.

- **Bug Fixes**
	- Corrected the path for sourcing the token in the `deleteAllBundles.sh` script.

- **Removals**
	- Removed several outdated scripts, including `loadAllBundles.sh`, `getConnectedUsers.sh`, and `createAllPerimeter.sh`, which improves overall project clarity and reduces redundancy.

### CODE REVIEW

File: src/test/resources/deleteAllBundles.sh [hunk 1]
```diff
@@ -19,6 +19,6 @@ then
 fi
 
 echo "Will delete all bundles on $url"
-source ../getToken.sh $url
+source ./getToken.sh $url
 curl -s -X DELETE "$url:2100/processes" -H "Authorization:Bearer $token"
 echo ""
```
[the remark this answers is not in the document]
Comment (freddidierRTE) [reply src/test/resources/deleteAllBundles.sh:24]: This script will be replaced in another issue

File: src/test/resources/loadTestConf.sh [hunk 2]
```diff
@@ -18,18 +18,8 @@ then
 	url="http://localhost"
 fi
 (
-	source ./getToken.sh $url
-	cd bundles
+	opfab login $url 2002 admin test
 	./deleteAllBundles.sh $url
```
[the remark this answers is not in the document]
Comment (freddidierRTE) [reply src/test/resources/loadTestConf.sh:22]: This script is only use for dev to load test data , there is no security risk or issue here

File: src/test/resources/loadTestConf.sh [hunk 3]
```diff
@@ -18,18 +18,8 @@ then
...
-	cd ../perimeters
-	./createAllPerimeter.sh
-	cd ../processMonitoring
-	opfab processmonitoring load processMonitoring.json
-	cd ../realTimeScreens
-	opfab realtimescreen load realTimeScreens.json
-	cd ../businessData
-	./loadAllBusinessData.sh
+	opfab bundle load bundles/*
+	opfab commands loadTestConf.txt
```
[the remark this answers is not in the document]
Comment (freddidierRTE) [reply src/test/resources/loadTestConf.sh:24]: This script is only used for dev with fixed login/pwd , no need to have precise error handling

File: src/test/resources/getToken.sh [hunk 4]
```diff
@@ -31,7 +31,7 @@ fi
 getToken() {
 	echo "Get token for user $username on $url"
 	access_token_pattern='"access_token":"([^"]+)"'
-	response=$(curl -v -s -X POST -d "username="$username"&password="$password"&grant_type=password" $url:2002/auth/token)
+	response=$(curl -s -X POST -d "username="$username"&password="$password"&grant_type=password" $url:2002/auth/token)
```
[the remark this answers is not in the document]
Comment (freddidierRTE) [reply src/test/resources/getToken.sh]: ok , done

File: src/test/resources/deleteAllBundles.sh [hunk 5]
```diff
@@ -19,6 +19,6 @@ then
 fi
 
 echo "Will delete all bundles on $url"
-source ../getToken.sh $url
+source ./getToken.sh $url
```
[the remark this answers is not in the document]
Comment (freddidierRTE) [reply src/test/resources/deleteAllBundles.sh:22]: it is present

**Review by vlo-rte — APPROVED** [review]

**Pull request merged**